contracts/protocore/GenesisSelfProtocore.sol

```diff
@@ -25,6 +25,21 @@ contract GenesisSelfProtocore {
 
     /* Storage */
 
+    /** Metachain id of the auxiliary chain. */
+    bytes32 public genesisAuxiliaryMetachainId;
```

Comment:
It should be actually `genesisSelfMetachainId` (otherwise SelfProtocore setup will fail), however, should we name it genesisMetachainId as it's already in GenesisSelfProtocore? If not, we should update also `genesisEpochLength` to `genesisSelfEpochLength` and so on.

Reply:
I was not able to understand this comment.
I updated the variable name to `genesisMetachainId`